django/db/backends/mysql/client.py

```diff
@@ -8,11 +8,11 @@ class DatabaseClient(BaseDatabaseClient):
     def settings_to_cmd_args_env(cls, settings_dict, parameters):
         args = [cls.executable_name]
         env = None
-        db = settings_dict['OPTIONS'].get('db', settings_dict['NAME'])
+        database = settings_dict['OPTIONS'].get('database', settings_dict['NAME'])
         user = settings_dict['OPTIONS'].get('user', settings_dict['USER'])
         password = settings_dict['OPTIONS'].get(
             'password',
-            settings_dict['OPTIONS'].get('passwd', settings_dict['PASSWORD'])
+            settings_dict['OPTIONS'].get('password', settings_dict['PASSWORD'])
```

Comment:
This change is unnecessary and unrelated. We support both `password` and `passwd` options in `DATABASES['OPTIONS']`.